src/Spinner/Spinner.js

```diff
@@ -243,5 +246,13 @@ module.exports = kind(
 	* @see enyo/AccessibilitySupport~AccessibilitySupport#accessibilityLive
 	* @public
 	*/
-	accessibilityLive: 'off'
+	accessibilityLive: 'off',
+
+	/**
+	* @default Loading
+	* @type {String}
+	* @translated loading message
+	* @public
+	*/
+	loadingMessage: $L('Loading')
```

Comment:
If you intended to use this as a constant that can be used when set content no app side, this can be not a variable but a constant. So, I recommend to explicitly export the constant on Spinner module like below. You can reference LightPanels code that exporting Direction.

```
var
	$L = require('../i18n'),
	loadingString = $L('Loading');

mosule.exports.LOADING_STRING = loadingString;
```